README.md

```diff
@@ -3,53 +3,57 @@
 --->
 <!---Short description of what the extension allows the user to do and key features in bullet points below 
 --->
-The IBM Blockchain extension allows a user to write and test chaincode projects in Visual Studio Code:
-* Generate skeleton chaincode projects
+The IBM Blockchain extension allows a user to easily write and test smart contract projects in Visual Studio Code:
```

Comment:
"IBM Blockchain Platform" extension will be the official name.  

Also, I dislike a README telling me something is easy, as it immediately makes me think "I'LL BE THE JUDGE OF THAT, THANK YOU!" but that's more of an opinion ;)